climada/entity/exposures/base.py

```diff
@@ -700,6 +709,11 @@ def write_raster(self, file_name, value_name='value', scheduler=None):
             raster, meta = u_coord.points_to_raster(self, [value_name], scheduler=scheduler)
             u_coord.write_raster(file_name, raster, meta)
 
+    def _replace_if_by_impf(self):
+        LOGGER.info('Replacing if_ by impf_ in Exposure geodataframe')
```

Comment:
I suggest being more explicit here.
```suggestion
        LOGGER.warning(" 'if_'  was replaced by 'impf_' in the Exposure's GeoDataFrame. The name 'if_' is deprecated. ")
```